<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>670</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>591</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="var1">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="var2">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="var3">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="var4">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
